setup.cfg

```diff
@@ -27,11 +27,17 @@ zip_safe = False
 install_requires =
     click>=7.0.0
     click-option-group>=0.5.3
-    invenio-records-marc21[postgresql,elasticsearch7]>=0.1.0
+    invenio-records-marc21[elasticsearch7]>=0.4.0
     requests>=2.0.0
+    # TODO: Prevent build fail remove after it is fixed in base module
```

Comment:
is it possible to remove that? What was the reason to add those 4 imports?

Reply:
hm, it seams those commits are done another time. we should fix those things with another PR